backend_new/src/utilities/build-filter.ts

```diff
@@ -34,29 +36,39 @@ export function buildFilter(
         .split(',')
         .map((s) => s.trim().toLowerCase())
         .filter((s) => s.length !== 0),
-      mode: Prisma.QueryMode.insensitive,
+      mode: caseSensitive
+        ? Prisma.QueryMode.default
+        : Prisma.QueryMode.insensitive,
```

Comment:
This is repeated a lot in this file. I'd recommend changing the `caseSensitive` value on line 27 to be this instead
```
const caseSensitive = filter.caseSensitive ? Prisma.QueryMode.default
        : Prisma.QueryMode.insensitive
```

Reply:
@YazeedLoonat reminder here!

Reply:
@YazeedLoonat , please update with @ludtkemorgan's suggestion.